plugins/copr.py

```diff
@@ -482,9 +483,19 @@ def _download_repo(self, project_name, repo_filename):
                     import urllib
                     res = urllib.urlopen(self.copr_url + "/coprs/" + project_name)
                     status_code = res.getcode()
-                if str(status_code) != '404':
-                    raise dnf.exceptions.Error(_("This repository does not have"
-                                                 " any builds yet so you cannot enable it now."))
+                if status_code != 404:
+                    error_msg = "It wasn't possible to enable this project.\n"
```

Comment:
Can we print some more info we know, like what is the owner, project and chroot?